src/SFA.DAS.CaptureTheFlag.Web/Startup.cs

```diff
@@ -27,13 +30,18 @@ public void ConfigureServices(IServiceCollection services)
             services.AddDbContext<ApplicationDbContext>(options =>
                 options.UseSqlServer(
                     Configuration.GetConnectionString("GameDbConnection")));
-            services.AddDefaultIdentity<IdentityUser>(
+
+            //services.AddDefaultIdentity<IdentityUser>(
+            //        options => options.SignIn.RequireConfirmedAccount = true)
+            //    .AddEntityFrameworkStores<ApplicationDbContext>();
+            services.AddDefaultIdentity<ApplicationUser>(
                     options => options.SignIn.RequireConfirmedAccount = true)
                 .AddEntityFrameworkStores<ApplicationDbContext>();
             services.AddControllersWithViews();
             services.AddSingleton<IGameRepository>(new GameRepository());
             services.AddSignalR();
             services.AddRazorPages();
+            services.AddScoped<IForumService, ForumService>();
```

Comment:
Although with DI scopes there is never a "right" answer I think you chose the most appropriate here. AddScoped means objects are created once per (web) request, the alternative of AddTransient which you could also have used would create a new instance on every injection.